benchmark/src/androidTest/java/com/apollographql/apollo/benchmark/Utils.kt

```diff
@@ -0,0 +1,51 @@
+import androidx.test.platform.app.InstrumentationRegistry
+import com.apollographql.apollo.api.CustomScalarAdapters
+import com.apollographql.apollo.api.Operation
+import com.apollographql.apollo.api.ResponseField
+import com.apollographql.apollo.api.internal.ResolveDelegate
+import com.apollographql.apollo.benchmark.test.R
+import com.apollographql.apollo.cache.normalized.CacheKey
+import com.apollographql.apollo.cache.normalized.Record
+import com.apollographql.apollo.cache.normalized.internal.RealCacheKeyBuilder
+import com.apollographql.apollo.cache.normalized.internal.ResponseNormalizer
+import com.apollographql.apollo.internal.response.RealResponseWriter
+import com.apollographql.apollo.response.OperationResponseParser
+import okio.BufferedSource
+import okio.buffer
+import okio.source
+
+object Utils {
+  fun bufferedSource() = InstrumentationRegistry.getInstrumentation().context.resources.openRawResource(R.raw.largesample)
+      .source()
+      .buffer()
+
+  private val responseNormalizer = object : ResponseNormalizer<Map<String, Any?>>() {
+    override fun cacheKeyBuilder() = RealCacheKeyBuilder()
+
+    override fun resolveCacheKey(field: ResponseField, record: Map<String, Any?>) = CacheKey.NO_KEY
+  }
+
+  fun <D : Operation.Data> computeRecordsDuringParsing(operation: Operation<D>, bufferedSource: BufferedSource): Collection<Record?>? {
```

Comment:
nit: should we rename it to more explicit like `parseWithInlinedNormilizer`?
Also I don't think we need to return records, as we don't use it?

Reply:
Works for me 👍